lib/pengine/clone.c

```diff
@@ -528,7 +528,7 @@ clone_print(resource_t * rsc, const char *pre_text, long options, void *print_da
     if(rsc->variant == pe_master) {
         enum rsc_role_e role = configured_role(rsc);
 
-        if(role > RSC_ROLE_STOPPED && role < RSC_ROLE_MASTER) {
+        if(role == RSC_ROLE_SLAVE) {
             short_print(list_text, child_text, "Slaves (target-role)", NULL, options, print_data);
```

Comment:
It's correct -- the only possibilities >stopped and <master are Started or Slave, and as Lars mentioned, target-role:Started doesn't constrain us in any way, so there's no need to print it in that case.